assets/src/scss/layout/_country-selector.scss

```diff
@@ -28,7 +28,7 @@
 
   &.open {
     .country-selector-toggle-container {
-      border-bottom: 1px solid transparentize($white, 1);
+      border-bottom: 1px solid rgba(var(--white), 1);
```

Comment:
Not sure why we use `transparentize` here (or `rgba`) since opacity is 1, seems we could just use the color directly 🤔